.github/workflows/helm_build_release.yaml

```
@@ -3,16 +3,31 @@ name: Build and release helm chart
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - '.devcontainer/**'
+      - '.github/**'
+      - '.test/**'
+      - 'README.md'
+      - 'LICENSE'
 
 jobs:
   helm-release:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
+      - name: Get chart version
+        id: get_chart_version
+        uses: Energinet-Datahub/.github/.github/actions/yaml-get@main
+        with:
+          yaml_file: chart/Chart.yaml
+          yaml_path: version
+
       - name: Release helm chart
-        uses: Energinet-DataHub/.github/.github/actions/helm-release-chart@helm_releases_action
+        uses: Energinet-DataHub/.github/.github/actions/helm-release-chart@5.0.0
```

5.0.0 is not latest on the .github repo, and currently there is not always created a tag. 
Alternaly hash could be used, but for now i suggest main.
```suggestion
        uses: Energinet-DataHub/.github/.github/actions/helm-release-chart@main
```